infrastructure/terraform/index-service.tf

```diff
@@ -22,7 +22,7 @@ module "index_secrets" {
     MONITORING_ENABLE        = "false"
     PUBLISH_METADATA         = "false"
     AZURE_ACCOUNT            = "${azurerm_storage_account.base.name}"
-    PULSE_USERNAME           = "${module.events_rabbitmq_user.username}"
```

Comment:
geez, lol

Reply:
Yeah, this has convinced me yet more that we should generate these configs rather than copy-pasting everywhere.